ca_gazebo/launch/create_2.launch

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="cmd_vel" default="/cmd_vel" />
+  <remap from="/cmd_vel" to="$(arg cmd_vel)"/>
+  <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find ca_description)/urdf/create_2.urdf.xacro'" />
```

Comment:
The robot description and state publisher already exist in ca_description. We can replace L5-6 with:

```
<include file="$(find ca_description)/launch/create_2.launch" />
```